$MirOS: ports/plan9/kencc/patches/patch-utils_c2l_cc_h,v 1.1 2006/03/31 03:50:19 tg Exp $
--- utils/c2l/cc.h.orig	Thu Mar 30 23:54:27 2006
+++ utils/c2l/cc.h	Thu Mar 30 23:55:24 2006
@@ -565,7 +565,7 @@ void*	allocn(void*, long, long);
 void*	alloc(long);
 void	cinit(void);
 int	compile(char*, char**, int);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 int	filbuf(void);
 int	getc(void);
 long	getr(void);
